docs/UserGuide.md

```diff
@@ -156,7 +156,7 @@ Examples:
 
 ### Finding a person: `find`
 
-Format: ‘find n/NAME p/PHONE_NUMBER r/ROOM_NUMBER t/Tags’’
+Format: ‘find n/NAME p/PHONE_NUMBER r/ROOM_NUMBER t/TAG’
```

Comment:
Should the command be in backticks (``) instead of '' to standardise with the rest?